apps/teams-test-app/src/components/privateApis/FilesAPIs.tsx

```diff
@@ -1,9 +1,46 @@
 import { FileOpenPreference, files, SdkError } from '@microsoft/teams-js';
-import React, { ReactElement } from 'react';
+import React, { ChangeEvent, ReactElement } from 'react';
 
+import { noHostSdkMsg } from '../../App';
 import { ApiWithCheckboxInput, ApiWithoutInput, ApiWithTextInput } from '../utils';
+import { ApiContainer } from '../utils/ApiContainer';
 import { ModuleWrapper } from '../utils/ModuleWrapper';
 
+export const FileUpload: React.FC = () => {
+  const fileUploadName = 'fileUpload';
+  const [result, setResult] = React.useState<string>('');
+  const onChangeCallback = React.useCallback(
+    (event: ChangeEvent<HTMLInputElement>) => {
+      setResult(noHostSdkMsg);
```

Comment:
with this, will it show that "no sdk response" message first then flip one of the results? (line 22, or 25_

Reply:
yes, that is correct.

Reply:
just for my curiosity, is it any specific reason we need this default?

Reply:
It essentially states out that API was called from Teams test app successfully but did not get any response. There are cases when API call is halted at test app level eg. when the param is not valid